pkg/machine/qemu/config.go

```diff
@@ -43,6 +43,15 @@ func findQEMUBinary() (string, error) {
 	return execPath, nil
 }
 
+func setQMPMonitorSocket(m *MachineVM) error {
```

Comment:
Same here w.r.t. IN/OUT arguments.

Rather turn it into a `func (m *MachineVM) setFooBar() error`